slice/IceBox/ServiceManager.ice

```diff
@@ -63,7 +63,7 @@ module IceBox
         void stopService(string service)
             throws AlreadyStoppedException, NoSuchServiceException;
 
-        /// Registers a new observer with this ServiceManager.
+        /// Registers a new observer with this service manager.
```

Comment:
ServiceManager gets autolinked to C++ Ice::ServiceManager that we no longer include in the doc since the user doesn't implement this generated interface.

And, once we fix autolink to link to proxies, it won't make sense to have a ServiceManagerPrx here.